include/ck_tile/ops/fmha/pipeline/block_fmha_bwd_dq_dk_dv_pipeline_trload_qr_qtr_dor.hpp

```diff
@@ -752,8 +752,7 @@ struct BlockFmhaBwdDQDKDVPipelineTrLoadQRQTRDOR
                                    dq_acc);
         else
             tile_elementwise_inout([&raw_scale](auto& x) { x = x * raw_scale; }, dq_acc);
-        // static_assert(kIsDeterministic);
-        dq_epilogue(dq_dram_window, dq_acc);
+        dq_epilogue(dq_dram_window, dq_acc, nullptr);
```

Comment:
(1) do we have to pass p_smem to cshuffle epilogue? if not, e.g. declare it as __shared__ inside operator(), we might be able to have a default value for dram windows and avoid this mess with signature changes
(2) maybe we can retain the old signature in future having operator() overload instead of changing the signature